src/Tasks/Microsoft.NET.Build.Tasks/RunReadyToRunCompiler.cs

```diff
@@ -102,5 +220,25 @@ protected override int ExecuteTool(string pathToTool, string responseFileCommand
 
             return base.ExecuteTool(pathToTool, responseFileCommands, commandLineCommands);
         }
+
+        #region TEMP - DELETE BEFORE MERGE (used for repros now)
```

Comment:
Note to delete this as you intended.

Reply:
FWIW, filed https://github.com/Microsoft/msbuild/issues/4315. I really think this should be fixed in msbuild